nc-news/src/components/Comments.jsx

```diff
@@ -1,3 +1,46 @@
-function Comments() {}
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchArticleComment } from "../api";
+
+function Comments() {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    fetchArticleComment(article_id).then((commentsFromAPI) => {
+      setComments(commentsFromAPI);
+    });
+  }, [article_id]);
+
+  return (
+    <section>
+      <h2 id="CommentTitle">Comments</h2>
+      <ul>
+        {comments.map((comment) => {
+          const createdAt = new Date(comment.created_at);
+          //console.log(createdAt);
+          const time = new Date(createdAt).toLocaleTimeString("en", {
+            timeStyle: "short",
+            hour12: false,
+            timeZone: "GMT",
+          });
+          const day = createdAt.getDate();
+          const year = createdAt.getFullYear();
+          const month = createdAt.getMonth();
```

Comment:
Love this date formatting. Consider extracting the functionality to a util function to make your render a little neater (plus you can re-use it in the future)